.gitignore

```diff
@@ -31,6 +31,8 @@
 /example/*/out
 .idea
 *.iml
+*.ipr
+*.iws
 
```

Comment:
I didn't found any files with such pattern under current code base.

Reply:
Found some doc about the file format: https://www.jetbrains.com/help/idea/creating-and-managing-projects.html

Reply:
They are the config files generated for me when I run "./gradlew idea" :D

Reply:
> They are the config files generated for me when I run "./gradlew idea" :D

We should probably want to make `gradlew idea` throw exception. Native IDEA project import should be preferred rather than `gradlew idea`

Reply:
I see, didn't realize the native import is the preferred way and there is already 'idea-ext' in the project.
Closing this PR :)

Reply:
Well, if you happen to have ipr and iws files, we can merge the PR.
However, I'm using IDEA, and I find it is easier to use regular project import rather than trying to use other approaches.